plugins/jobs/web_client/js/views/JobListWidget.js

```diff
@@ -1,16 +1,19 @@
 girder.views.jobs_JobListWidget = girder.View.extend({
     events: {
-        'click .g-job-trigger-link': function (e) {
+        'click .g-job-checkbox-menu input': function (e) {
             var cid = $(e.target).attr('cid');
             this.trigger('g:jobClicked', this.collection.get(cid));
         }
     },
 
     initialize: function (settings) {
+        var self = this;
```

Comment:
Can you use `_.bind(function (e) {}, this);` instead of `self = this`?  That is more consistent with Girder js generally.